docs/specs/verification.md

```diff
@@ -0,0 +1,66 @@
+# User Verification
+
+## Definitions
+
+A user is verified if the user fulfill the condition specified by criteria.
+
+An identity is verifiable if:
+- it is an SSO identity; or
+- it is a login ID identity with a configured verifiable login ID key.
+
+An identity is verified if:
+- it is an SSO identity; or
+- it is a login ID identity, and the user has a matching OOB-OTP authenticator.
+
+## Criteria
+
+Developer can configure the criteria used to determine verification status
+of a user. There are two possible criteria:
+
+- `any`: User has a verifiable identity and one of user's verifiable identities is verified.
```

Comment:
`User has at least one verifiable identity and at least one verified verifiable identity.`